tests/end-to-end/external-solution-integration/internal/scenario/e2e/e2e.go

```diff
@@ -1,71 +0,0 @@
-package e2e
```

Comment:
When we delete this, are you aware that we are deleting the external solution test for the compatibility layer?

Reply:
see https://github.com/kyma-project/kyma/pull/7738#discussion_r396640059